lib/index.js

```diff
@@ -30,6 +30,12 @@ var GTM = module.exports = integration('Google Tag Manager')
  */
 
 GTM.prototype.initialize = function() {
+  // If window.dataLayer already exists and isn't an array, fail gracefully.
+  if (window.dataLayer && !Array.isArray(window.dataLater)) {
+    console.error('window.dataLayer already exists - not running Segment<>GTM destination')
```

Comment:
I think we can do this.debug - https://github.com/segment-integrations/analytics.js-integration-bronto/blob/391bf8bbbedb7d63e1104fc4ac8821daa00a766d/lib/index.js#L37